bitnami/etcd/README.md

```diff
@@ -102,6 +102,9 @@ The following tables lists the configurable parameters of the etcd chart and the
 | `persistence.accessMode`                          | PVC Access Mode for etcd volume                                                                                                                           | `ReadWriteOnce`                                             |
 | `persistence.size`                                | PVC Storage Request for etcd volume                                                                                                                       | `8Gi`                                                       |
 | `persistence.annotations`                         | Annotations for the PVC                                                                                                                                   | `{}`                                                        |
+| `podDisruptionBudget.enabled`                     | Pod Disruption Budget toggle                                                                                                                              | `false`                                                     |
+| `podDisruptionBudget.minAvailable`                | Minimum available pods                                                                                                                                    | `nil`                                                       |
+| `podDisruptionBudget.maxUnavailable`              | Maximum unavailable pods                                                                                                                                  | `nil`                                                       |
```

Comment:
Can I suggest using `pdb` instead of `podDisruptionBudget` for these parameters? We're using this on other charts and it's more shorter and easier to manage